vignettes/set_up.Rmd

```diff
@@ -0,0 +1,23 @@
+---
+title: "Additional steps for setting up workflows for your R package"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Additional steps for setting up workflows for your R package}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Some workflows in `{ghactions4r}` take additional setup steps besides running the `use_*` function. Here is a guide on the additional steps to be taken.
+
+## Setting up code coverage for your R package
+
+1) Clone your R package locally.
+2) In the local clone, run [ghactions4r::use_calc_coverage()](https://nmfs-fish-tools.github.io/ghactions4r/reference/use_calc_coverage.html) to generate the the github action file or any other files needed. Change the build triggers if desired.
+3) Commit the files and push them up to GitHub. See if a github action runs. If it does not, try pushing to your main branch to trigger a run, if your build triggers include a run on pushing to the main branch (the default).
+4) Set up an account on [codecov](https://about.codecov.io/) or [coveralls](https://coveralls.io/). For codecov, [create an account](https://docs.codecov.com/docs/github-2-getting-a-codecov-account-and-uploading-coverage#create-a-codecov-account), then install the [github app](https://docs.codecov.com/docs/github-2-getting-a-codecov-account-and-uploading-coverage#install-the-github-app-integration). For coveralls, [sign up to use OAuth through GitHub](https://coveralls.io/sign-up).

```

Comment:
Question: Is an account required to submit a codecov report? I did not find codecov under my [GitHub authorized apps](https://github.com/settings/applications), but I was able to push a codecov report online using GitHub Actions (see report [here](https://app.codecov.io/gh/nmfs-fish-tools/ghBackup/tree/calc-codecov)). Could it be that the report is uploaded through a token?

Reply:
Good point, I think it worked because we have codecov as an app under the nmfs-fish-tools organization. Perhaps I should note that if you are in an organization that has already installed the app, that you don't need an account?

Reply:
Sounds good!